src/main/java/com/trikzon/flora_doubling/FloraDoubling.java

```diff
@@ -44,7 +41,8 @@
     public static Config.ConfigBean CONFIG = new Config.ConfigBean();
 
     boolean isTargetFlower(Block block) {
-        return (DOUBLING_FLORA_TAG.contains(block) || CONFIG.doublingFlora.contains(getId(block.asItem())) ||
+        return ((block.getClass() != WitherRoseBlock.class) || CONFIG.allowWitherRoses) &&
```

Comment:
Maybe I'm wrong, but wouldn't it be better to do `block != Blocks.WITHER_ROSE` instead?

Reply:
@Trikzon I guess if they're the same class instance that'd work.